frontend/src/views/ConfirmationView.vue

```diff
@@ -129,6 +157,7 @@ export default {
       const payload = {
         isOk: this.confirmation.ok === 'yes',
         comment: this.confirmation.comment,
+        project: this.confirmation.projects,
```

Comment:
Here it's `this.confirmation.projects` but in line 174 it's `this.confirmation.project` (no `s`).